src/career/less/career.less

```diff
@@ -75,7 +75,7 @@
 
 .jobList {
   display: grid;
-  grid-template-rows: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
```

Comment:
As you can see in the images, this will result in the first 4 elements in the list having the exact same size; the size the will fit the largest of the first 4 elements, while the rest will have the size they should have. This is because `1fr 1fr 1fr 1fr` will create 4 boxes of equal size, to fit the largest, no matter how many elements are actually present in the DOM. Example:
![career-filler-grid](https://user-images.githubusercontent.com/14319313/48968478-8fe2e000-eff0-11e8-8542-d8f82797a941.png)
Note that this will also create all of that whitespace on mobile, which will just be a lot of whitespace before the footer.


To resolve this, we could hack it a bit, and force the size of the first 4 elements to fit their own min-height:
``` less
grid-template-rows: repeat(4, min-content);
```
Since the content is nothing when there are no extra elements, their height will be nothing.

Still a bit hacky, but it will show as it is supposed to. But if the filter bar is made larger, the problem will still arise again. If you find a better solution, I'm up for suggestions 😄

Reply:
Seems like that way seems to work fine. I didnt get that problem when the filter bar gets bigger. It seems to look just fine. if a whitespace issue on mobile ever becomes a problem we could always just use @media. Maybe a pic of what happends when the filter bar gets bigger

Reply:
Seems I was wrong about expanding the height of the filter menu, it did not result in any issues for me when testing. I think we will have to keep this thing in mind when the internship application season starts after the summer, to look out for potential problems.